kute-android-app/app/build.gradle

```diff
@@ -2,7 +2,7 @@ apply plugin: 'com.android.application'
 
 
 android {
-    compileSdkVersion 27
+    compileSdkVersion 26
     buildToolsVersion '27.0.2'
```

Comment:
Compile SDK should be the latest version. Besides compiled SDK version and buildToolsVersion should be the same.

Reply:
oh ok.
i will convert it to version 27.